utils/models/__init__.py

```diff
@@ -0,0 +1,90 @@
+from .skip import skip
+from .texture_nets import get_texture_nets
+from .resnet import ResNet
+from .unet import UNet
+
+import torch.nn as nn
+import torch
+
+
+def get_net(
+    input_depth,
+    NET_TYPE,
```

Comment:
Don't use capitalize variable names, by conventions, they are reserved for constants.
```suggestion
    net_type,
```